application/controllers/gnu_encryptor.py

```diff
@@ -0,0 +1,31 @@
+import gnupg
+import gnupg._parsers
+import json
+import logging
+from structlog import wrap_logger
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+class GNUEncrypter:
+
+    def __init__(self, public_key, passphrase=None, always_trust=True):
+        self.gpg = gnupg.GPG()
+        self.gpg.import_keys(public_key.encode('utf-8'))
+        gnupg._parsers.Verify.TRUST_LEVELS["ENCRYPTION_COMPLIANCE_MODE"] = 23
+
+    def encrypt(self, payload, recipient):
+        """
+        Encrypts the payload using the recipient values
+
+        :param payload: the value to encrypt
+        :param recipient: who is it for
+        :return: string of encrypted data
+        """
+        enc_data = self.gpg.encrypt(json.dumps(payload).encode('utf-8'), recipient, always_trust=True)
```

Comment:
We may need to change this because we're not going to be converting the spreadsheet to a JSON object anymore. The idea will be to encrypt the file as is with GPG and send that directly to the bucket.

Therefore you may need to remove the call to json.dumps and change how this function is called